scripts/west_commands/sdk.py

```diff
@@ -0,0 +1,528 @@
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            epilog=textwrap.dedent(
+                """
+            Installing SDK:
+
+                Run 'west sdk install' to install Zephyr SDK.
+
+                Set --version option to install a specific version of the SDK.
+                If not specified, the install version is detected from "${ZEPHYR_BASE}/SDK_VERSION file.
+                SDKs older than 0.14.1 are not supported.
+
+                You can specify the installation directory with --install-dir.
+                If the specified version of the SDK is already installed,
+                the already installed SDK will be used regardless of the settings of
+                --install-dir.
+
+                Typically, Zephyr SDK archives contain only one directory named zephyr-sdk-<version>
+                at the top level.
+                The SDK archive is extracted to the home directory if --install-dir is not specified.
+                In this case, SDK will install into ${HOME}/zephyr-sdk-<version>.
+                If --install-dir is specified, the `zephyr-sdk-<version>` will be renamed and relocated to
+                the specified location.
+
+                --interactive, --toolchains, --no-toolchains and --no-hosttools options
+                specify the behavior of the installer. Please see the description of each option.
+
+                --personal-access-token specifies the GitHub personal access token.
+                This helps to relax the limits on the number of REST API calls.
+
+                --api-url specifies the REST API endpoint for GitHub releases information
+                when installing the SDK from a different GitHub repository.
+            """
+            ),
+        )
+
+        install_args_parser.add_argument(
+            "-v",
+            "--version",
+            default=None,
+            nargs="?",
+            metavar="SDK_VER",
+            help="version of the Zephyr SDK to install. "
+            "If not specified, the install version is detected from "
+            "${ZEPHYR_BASE}/SDK_VERSION file.",
+        )
+        install_args_parser.add_argument(
+            "-d",
+            "--install-dir",
+            default=None,
+            metavar="DIR",
+            help="SDK install destination directory.",
+        )
+        install_args_parser.add_argument(
+            "-i",
+            "--interactive",
+            action="store_true",
+            help="launches installer in interactive mode. "
+            "--toolchains, --no-toolchains and --no-hosttools will be ignored if this option is enabled.",
+        )
+        install_args_parser.add_argument(
+            "-t",
+            "--toolchains",
+            metavar="toolchain_name",
+            nargs="+",
+            help="toolchain(s) to install (e.g. 'arm-zephyr-eabi'). "
+            "If this option is not given, all toolchains will be installed.",
```

Comment:
SDK 0.15.2 and later versions have the sdk_toolchains file, so if this file is present, I made the output of `sdk list` display toolchains that are not installed.
I have decided that displaying help is unpreferable because it requires retrieving information via the network.